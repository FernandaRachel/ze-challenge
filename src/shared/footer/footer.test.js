import { getByTestId } from '@testing-library/dom';
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer';

describe('Footer Snapshot', () => {
    test('snapshot renders', () => {
        const arr = [];
        const component = renderer.create(<Footer />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
describe('Checking footer', () => {

    test('Have footer class', () => {
        const arr = [];
        const component = renderer.create(<Footer />);
        let tree = component.toJSON();
        console.log('tree');
        console.log(tree);
        expect(tree.props.className).toContain('footer')
    });
});

