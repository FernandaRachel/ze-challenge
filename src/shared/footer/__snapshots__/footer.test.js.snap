// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot snapshot renders 1`] = `
<div
  className="footer"
  data-testid="main-div"
/>
`;

exports[`Footer snapshot renders 1`] = `
<div
  className="footer"
  data-testid="main-div"
/>
`;
